client/templates/topic/topicInfoItemList.html

```diff
@@ -130,6 +130,11 @@ <h4 class="panel-title">
                                         {{#if isDetailNew this.createdInMinute}}
                                              <span class="badge">New</span>
                                         {{/if}}
+
+                                        <!-- New-Topic-Label -->
```

Comment:
It seems that this comment is not correct. Since it follows the "New-Label" of the details, right? I would suggest to simply remove this comment ;-)